.github/workflows/pull-request.yaml

```diff
@@ -6,92 +6,10 @@ on:
       - main
 
 jobs:
-  lib-check:
-    name: Check libraries
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Check libs
-        uses: canonical/charming-actions/check-libraries@2.0.0-rc2
-        with:
-          credentials: "${{ secrets.CHARMHUB_TOKEN }}"
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-  static-lib:
-    name: Static analysis of /lib for Python 3.5
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Set up Python 3.5
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.5
-      - name: Install dependencies
-        run: python3 -m pip install tox
-      - name: Run static analysis for /lib for 3.5
-        run: tox -vve static-lib
-
-  static-charm:
-    name: Static analysis of the charm and tests
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Install dependencies
-        run: python3 -m pip install tox
-      - name: Run static analysis (charm)
-        run: tox -vve static-charm
-      - name: Run static analysis (unit tests)
-        run: tox -vve static-unit
-      - name: Run static analysis (integration tests)
-        run: tox -vve static-integration
-
-  lint:
-    name: Lint
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Setup Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Install dependencies
-        run: python3 -m pip install tox
-      - name: Run linters
-        run: tox -vve lint
-
-  unit-test:
-    name: Unit tests
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Install dependencies
-        run: python -m pip install tox
-      - name: Run tests
-        run: tox -e unit
-
-  integration-test:
-    name: Integration tests (microk8s)
-    uses: ./.github/workflows/integration-tests.yml
+  pull-request:
+    uses: canonical/observability/.github/workflows/pull-request.yaml@main
```

Comment:
Should this file be present in [this directory](https://github.com/canonical/observability-libs/tree/main/.github/workflows)?
🤔

Reply:
No, `observability` - not `observability-libs` :)

Reply:
small detail! ;-)